pyvista/utilities/fileio.py

```diff
@@ -569,6 +326,10 @@ def read_exodus(
 def read_plot3d(filename, q_filenames=(), auto_detect=True, attrs=None, progress_bar=False):
     """Read a Plot3D grid file (e.g., grid.in) and optional q file(s).
 
+    .. deprecated:: 0.35.0
+        This function is deprecated and will be removed in a future version.
+        Use :class:`pyvista.Plot3DMetaReader`.
+
```

Comment:
If we're deprecating this then I wonder if we should do the same to `read_legacy()` after all.

And the updated code below (and emitted warning) uses MultiBlockPlot3Reader; shouldn't this deprecation note also point to that?

Reply:
> And the updated code below (and emitted warning) uses MultiBlockPlot3Reader; shouldn't this deprecation note also point to that?

Good catch!

Reply:
> If we're deprecating this then I wonder if we should do the same to `read_legacy()` after all.

You've convinced me to deprecate it instead, it isn't too much extra work.